ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -52,8 +52,8 @@ class GeneticAlgorithm(RavenSampled):
   """
   convergenceOptions = {'objective': r""" provides the desired value for the convergence criterion of the objective function
                         ($\epsilon^{obj}$). In essence this is solving the inverse problem of finding the design variable
-                         at a given objective value, i.e., convergence is reached when: $$ Objective = \epsilon^{obj}$$.
-                        \default{1e-6}, if no criteria specified""",
+                         at a given objective value, i.e., convergence is reached when: $$ Objective = \epsilon^{obj}$$
+                        default{1e-6}, if no criteria specified""",
```

Comment:
Please put back the `\` before default, this is recognized by our autodoc generator.